app/src/main/kotlin/com/wire/android/ui/common/bottomsheet/RichMenuBottomSheetItem.kt

```diff
@@ -0,0 +1,105 @@
+package com.wire.android.ui.common.bottomsheet
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import com.wire.android.ui.common.dimensions
+import com.wire.android.ui.theme.DEFAULT_WEIGHT
+import com.wire.android.ui.theme.wireColorScheme
+import com.wire.android.ui.theme.wireTypography
+import io.github.esentsov.PackagePrivate
+
+@Composable
+fun RichMenuBottomSheetItem(
+    title: String,
+    subLine: String,
+    icon: (@Composable () -> Unit)? = null,
```

Comment:
i think we can pass here a default {} instead of null in action it makes sense to pass a null as we are adding a Column wrapping the action, but again up to you :P a small thing

Reply:
I also think we should pass a non nullable icon. Then we should check with the `state` parameter whether it makes sense to render it or not.